### ISSUE TITLE
Incorrect rendering of X and O

### ISSUE THREAD

**Issue by 0xDEADC0DER** [thomas-mauran/tic-tac-toe#4]
The hardcoded X and O ASCII arts don't render properly on terminals having large font size.

**Comment by thomas-mauran** [comment]
Yeah, having the same issue on [chess-tui](https://github.com/thomas-mauran/chess-tui) and to fix that you need to manualy zoom in or out of your terminal using ctrl + / - for the moment it is the only effective way to adjust the pieces size

**Comment by 0xDEADC0DER** [comment]
I think using the [Canvas widget](https://docs.rs/ratatui/latest/ratatui/widgets/canvas/struct.Canvas.html) might help.
The widget allows you to draw arbitrary shapes on the terminal.
Perfect for drawing diagonal lines and circles which are the required shapes for the game.
I am learning ratatui myself, maybe some time later I'll make a PR fixing this problem.

Same goes for [chess-tui](https://github.com/thomas-mauran/chess-tui)

**Comment by thomas-mauran** [comment]
Good idea, if you wanna make a pr go ahead !
